### PR TITLE
Add ability to show only a mesh wireframe with the wireframe filter

### PR DESCRIPTION
**Pull request by asnt**
Follow-up on #1689

```python
wireframe_filter.wireframe_only = True
```
wireframe_no_interior

```python
wireframe_filter.wireframe_only = False
```
wireframe_with_interior

### CODE REVIEW

**Comment by asnt** [comment]
> Thanks, this is really nice to have. Would you mind updating the test I had added to include a test of this new property?

Done.

File: vispy/visuals/tests/test_mesh.py [hunk 1]
```diff
@@ -81,4 +81,41 @@ def test_mesh_bounds():
...
+        pytest.raises(AssertionError, np.testing.assert_allclose,
+                      rendered_with_wf, rendered_wo_wf)
+
+        wireframe_filter.wireframe_only = True
+        rendered_with_wf_only = c.render()
+        # the result should be different from the two cases above
+        pytest.raises(AssertionError, np.testing.assert_allclose,
+                      rendered_with_wf_only, rendered_with_wf)
+        pytest.raises(AssertionError, np.testing.assert_allclose,
+                      rendered_with_wf_only, rendered_wo_wf)
```
Comment (djhoese) [inline vispy/visuals/tests/test_mesh.py:118]: This last test here is not raising an exception (they are apparently equal).

Comment (asnt) [reply vispy/visuals/tests/test_mesh.py:119]: Fixed. Thanks!

**Comment by djhoese** [comment]
This is great! Thanks. I love the simplicity of the test fix.

**Pull request merged**